user-guide/modules/ROOT/nav.adoc

```diff
@@ -1,11 +1,19 @@
 * xref:intro.adoc[]
-* xref:getting-started.adoc[]
 * xref:whats-inside.adoc[]
-* xref:reporting-issues.adoc[]
-* xref:advanced-builds.adoc[]
-* xref:faq.adoc[]
-* xref:tutorials.adoc[]
-* xref:testing-debugging.adoc[]
+* xref:getting-started.adoc[]
+
+* Building 
+** xref:installing-select-libraries.adoc[]
+** xref:advanced-builds.adoc[]
+
+* Using
+** xref:reporting-issues.adoc[]
```

Comment:
I wouldn't say "reporting issues" and "FAQ" are subcategories of "Using" boost.

Reply:
@alandefreitas I think they are part of the to and fro of using boost.

Reply:
I had another look at this and I think the problem is at another level. The category "Using" is always going to be ambiguous in the context of a "User Guide". Because "users" are always "using" it. This ends up implying "Using" is more like "Other", since everything is about "Using" but we chose to effectively hide these pages a little more. The solution would be to find a more descriptive name for this "Using" section then.